elixir/search/result.py

```diff
@@ -1,15 +1,24 @@
-from typing import Dict, List
+from typing import Dict, List, NamedTuple
+
+import torch
 
 from elixir.chunk import ChunkGroup
 
 
+class ChunkPlan(NamedTuple):
+    name_list: List[str]
```

Comment:
What is a ChunkPlan? Add some docstring for clarity.

Reply:
Done